config/locales/client.en.yml

```
@@ -3136,6 +3136,10 @@ en:
     safe_mode:
       enabled: "Safe mode is enabled, to exit safe mode close this browser window"
 
+    too_many_sessions:
+      one: You were logged in to more than %{limit} devices. The oldest %{count} session has been logged out. Review recently used devices in your <a href="%{url}">account preferences</a>.
```

```suggestion
      one: You were logged in to more than %{limit} devices. The oldest session has been logged out. You may review your recently used devices in your <a href="%{url}">account preferences</a>.
```
I don't think we need to write "The oldest 1 session" here.